App crashing after skia animation for both android and ios

### Description

I'm facing build time warnings for android like
/node_modules/@shopify/react-native-skia/android/cpp/jni/JniSkiaManager.cpp:12:29: warning: extra tokens at end of #ifdef directive [-Wextra-tokens]
#ifdef REACT_NATIVE_VERSION >= 75
                            ^
                           
and crash error like
2024-09-30 16:55:36.804 31256-31256 unknown:ReactModalHost  pid-31256                            E  Creating new dialog from context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 16:55:36.979 31256-31256 unknown:ReactModalHost  pid-31256                            E  Updating existing dialog with context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 16:56:05.461 31256-31309 ReactNativeJS           pid-31256                            E  'VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead.', '\n    at VirtualizedList (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:78791:36)\n    at FlatList (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:78095:36)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTScrollView\n    at ScrollView (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:69749:36)\n    at ScrollView\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at ImageBackground (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:120000:36)\n    at BackgroundImage (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:184177:24)\n    at LiveScreen (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:292540:98)\n    at StaticContainer (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:131568:17)\n    at EnsureSingleNavigator (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:127340:24)\n    at SceneView (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:131473:22)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at DebugContainer (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:204416:36)\n    at MaybeNestedStack (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:203969:23)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RNSScreen\n    at anonymous (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:75641:62)\n    at Suspender (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:206989:22)\n    at Suspense\n    at Freeze (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:206999:23)\n    at DelayedFreeze (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runMo
2024-09-30 16:56:05.572 31256-31256 unknown:ReactModalHost  pid-31256                            E  Creating new dialog from context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 16:56:05.620 31256-31256 unknown:ReactModalHost  pid-31256                            E  Updating existing dialog with context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 16:56:36.391 31256-31256 unknown:ReactModalHost  pid-31256                            E  Creating new dialog from context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 16:56:36.427 31256-31256 unknown:ReactModalHost  pid-31256                            E  Updating existing dialog with context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 16:56:39.074 31256-31256 libc                    pid-31256                            A  Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x100bf86c0 in tid 31256 (com.ecotricity), pid 31256 (com.ecotricity)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A  Cmdline: com.ecotricity
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A  pid: 31256, tid: 31256, name: com.ecotricity  >>> com.ecotricity <<<
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #01 pc 000000000085b020  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::JsiDomDeclarationNode::invalidateContext()+52) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #02 pc 000000000085b0d8  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::JsiDomDeclarationNode::onPropertyChanged(RNSkia::BaseNodeProp*)+32) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #03 pc 00000000007f1188  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::JsiDomNode::ensurePropertyContainer()::'lambda'(RNSkia::BaseNodeProp*)::operator()(RNSkia::BaseNodeProp*) const+84) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #04 pc 00000000007f1110  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #05 pc 00000000007f10a8  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (void std::__ndk1::__invoke_void_return_wrapper<void>::__call<RNSkia::JsiDomNode::ensurePropertyContainer()::'lambda'(RNSkia::BaseNodeProp*)&, RNSkia::BaseNodeProp*>(RNSkia::JsiDomNode::ensurePropertyContainer()::'lambda'(RNSkia::BaseNodeProp*)&, RNSkia::BaseNodeProp*&&)+48) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #06 pc 00000000007f1068  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #07 pc 00000000007efd90  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (std::__ndk1::__function::__func<RNSkia::JsiDomNode::ensurePropertyContainer()::'lambda'(RNSkia::BaseNodeProp*), std::__ndk1::allocator<RNSkia::JsiDomNode::ensurePropertyContainer()::'lambda'(RNSkia::BaseNodeProp*)>, void (RNSkia::BaseNodeProp*)>::operator()(RNSkia::BaseNodeProp*&&)+48) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #08 pc 00000000007ca160  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #09 pc 00000000007c9e5c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (std::__ndk1::function<void (RNSkia::BaseNodeProp*)>::operator()(RNSkia::BaseNodeProp*) const+60) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #10 pc 00000000007f7a30  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::NodeProp::updateValue(facebook::jsi::Runtime&, facebook::jsi::Value const&)+244) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #11 pc 00000000007f792c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::JsiDomNode::setProp(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)::'lambda'(RNSkia::NodeProp*)::operator()(RNSkia::NodeProp*) const+36) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #12 pc 00000000007f78e4  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #13 pc 00000000007f787c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (void std::__ndk1::__invoke_void_return_wrapper<void>::__call<RNSkia::JsiDomNode::setProp(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)::'lambda'(RNSkia::NodeProp*)&, RNSkia::NodeProp*>(RNSkia::JsiDomNode::setProp(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)::'lambda'(RNSkia::NodeProp*)&, RNSkia::NodeProp*&&)+48) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #14 pc 00000000007f783c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #15 pc 00000000007f664c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (std::__ndk1::__function::__func<RNSkia::JsiDomNode::setProp(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)::'lambda'(RNSkia::NodeProp*), std::__ndk1::allocator<RNSkia::JsiDomNode::setProp(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)::'lambda'(RNSkia::NodeProp*)>, void (RNSkia::NodeProp*)>::operator()(RNSkia::NodeProp*&&)+48) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #16 pc 00000000007f60a0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #17 pc 00000000007f5ba8  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (std::__ndk1::function<void (RNSkia::NodeProp*)>::operator()(RNSkia::NodeProp*) const+60) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #18 pc 00000000007f5888  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::NodePropsContainer::enumerateMappedPropsByName(std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&, std::__ndk1::function<void (RNSkia::NodeProp*)> const&)+276) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #19 pc 00000000007edf18  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (RNSkia::JsiDomNode::setProp(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+220) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #20 pc 00000000006911a8  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #21 pc 0000000000691034  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #22 pc 0000000000690f60  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (std::__ndk1::__bind_return<facebook::jsi::Value (RNJsi::JsiHostObject::*)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), std::__ndk1::tuple<RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1>, std::__ndk1::placeholders::__ph<2>, std::__ndk1::placeholders::__ph<3>, std::__ndk1::placeholders::__ph<4> >, std::__ndk1::tuple<facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&>, __is_valid_bind_return<facebook::jsi::Value (RNJsi::JsiHostObject::*)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), std::__ndk1::tuple<RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1>, std::__ndk1::placeholders::__ph<2>, std::__ndk1::placeholders::__ph<3>, std::__ndk1::placeholders::__ph<4> >, std::__ndk1::tuple<facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&> >::value>::type std::__ndk1::__bind<facebook::jsi::Value (RNJsi::JsiHostObject::* const&)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1> const&, std::__ndk1::placeholders::__ph<2> const&, std::__ndk1::placeholders::__ph<3> const&, std::__ndk1::placeholders::__ph<4> const&>::operator()<facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long>(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+164) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #23 pc 0000000000690e98  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #24 pc 0000000000690dec  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (facebook::jsi::Value std::__ndk1::__invoke_void_return_wrapper<facebook::jsi::Value>::__call<std::__ndk1::__bind<facebook::jsi::Value (RNJsi::JsiHostObject::* const&)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1> const&, std::__ndk1::placeholders::__ph<2> const&, std::__ndk1::placeholders::__ph<3> const&, std::__ndk1::placeholders::__ph<4> const&>&, facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long>(std::__ndk1::__bind<facebook::jsi::Value (RNJsi::JsiHostObject::* const&)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1> const&, std::__ndk1::placeholders::__ph<2> const&, std::__ndk1::placeholders::__ph<3> const&, std::__ndk1::placeholders::__ph<4> const&>&, facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+120) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #25 pc 0000000000690d50  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #26 pc 000000000068fb14  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/librnskia.so (std::__ndk1::__function::__func<std::__ndk1::__bind<facebook::jsi::Value (RNJsi::JsiHostObject::* const&)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1> const&, std::__ndk1::placeholders::__ph<2> const&, std::__ndk1::placeholders::__ph<3> const&, std::__ndk1::placeholders::__ph<4> const&>, std::__ndk1::allocator<std::__ndk1::__bind<facebook::jsi::Value (RNJsi::JsiHostObject::* const&)(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long), RNJsi::JsiHostObject*, std::__ndk1::placeholders::__ph<1> const&, std::__ndk1::placeholders::__ph<2> const&, std::__ndk1::placeholders::__ph<3> const&, std::__ndk1::placeholders::__ph<4> const&> >, facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+120) (BuildId: 52482bd67ca98f860a0b7130482be9555d5ec951)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #27 pc 0000000000299fc4  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #28 pc 0000000000299ea0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (std::__ndk1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long) const+132) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #29 pc 0000000000299df0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::DecoratedHostFunction::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+96) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #30 pc 0000000000299d18  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #31 pc 0000000000299c04  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::Value std::__ndk1::__invoke_void_return_wrapper<facebook::jsi::Value>::__call<facebook::jsi::DecoratedHostFunction&, facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long>(facebook::jsi::DecoratedHostFunction&, facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+136) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #32 pc 0000000000299b04  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #33 pc 0000000000297c60  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (std::__ndk1::__function::__func<facebook::jsi::DecoratedHostFunction, std::__ndk1::allocator<facebook::jsi::DecoratedHostFunction>, facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+136) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #34 pc 0000000000299fc4  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #35 pc 0000000000299ea0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (std::__ndk1::function<facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long) const+132) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #36 pc 0000000000299df0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::DecoratedHostFunction::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+96) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #37 pc 0000000000299d18  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #38 pc 0000000000299c04  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::Value std::__ndk1::__invoke_void_return_wrapper<facebook::jsi::Value>::__call<facebook::jsi::DecoratedHostFunction&, facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long>(facebook::jsi::DecoratedHostFunction&, facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+136) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #39 pc 0000000000299b04  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #40 pc 0000000000297c60  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (std::__ndk1::__function::__func<facebook::jsi::DecoratedHostFunction, std::__ndk1::allocator<facebook::jsi::DecoratedHostFunction>, facebook::jsi::Value (facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)>::operator()(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value const*&&, unsigned long&&)+136) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #41 pc 000000000009c7f0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #42 pc 00000000000ce538  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #43 pc 00000000000ed0b0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #44 pc 00000000000ec554  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #45 pc 00000000000ce830  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #46 pc 00000000000ce068  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #47 pc 00000000000ed02c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #48 pc 00000000000ec554  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #49 pc 00000000000ce830  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #50 pc 00000000000ce068  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #51 pc 00000000000ed02c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #52 pc 00000000000ec554  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #53 pc 00000000000ce830  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #54 pc 000000000016e5b0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #55 pc 00000000000ce538  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #56 pc 00000000000fafc4  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #57 pc 00000000000ed3e8  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #58 pc 00000000000ec554  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #59 pc 00000000000ce830  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #60 pc 00000000000ce068  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #61 pc 0000000000095060  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libhermes.so (BuildId: db6fd99979b1c632a8aa7fc3fb1374981dc2c7c4)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #62 pc 0000000000294490  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::RuntimeDecorator<facebook::jsi::Runtime, facebook::jsi::Runtime>::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+84) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #63 pc 00000000002915e0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::WithRuntimeDecorator<reanimated::ReanimatedReentrancyCheck, facebook::jsi::Runtime, facebook::jsi::Runtime>::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+96) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #64 pc 0000000000294490  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::RuntimeDecorator<facebook::jsi::Runtime, facebook::jsi::Runtime>::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+84) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #65 pc 00000000002a9d64  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::WithRuntimeDecorator<reanimated::AroundLock, facebook::jsi::Runtime, facebook::jsi::Runtime>::call(facebook::jsi::Function const&, facebook::jsi::Value const&, facebook::jsi::Value const*, unsigned long)+96) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #66 pc 0000000000236fe0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::Function::call(facebook::jsi::Runtime&, facebook::jsi::Value const*, unsigned long) const+108) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #67 pc 0000000000236e2c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::Function::call(facebook::jsi::Runtime&, std::initializer_list<facebook::jsi::Value>) const+116) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #68 pc 00000000002859b0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::Value facebook::jsi::Function::call<facebook::jsi::Value const&, facebook::jsi::Value&>(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value&) const+212) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #69 pc 000000000025b98c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jsi::Value reanimated::runOnRuntimeGuarded<facebook::jsi::Value&>(facebook::jsi::Runtime&, facebook::jsi::Value const&, facebook::jsi::Value&)+80) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #70 pc 000000000025b610  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (reanimated::NativeReanimatedModule::onRender(double)+220) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #71 pc 0000000000265e00  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #72 pc 0000000000265d50  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #73 pc 0000000000265c88  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #74 pc 0000000000265bd0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #75 pc 0000000000263d50  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #76 pc 0000000000418b58  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #77 pc 0000000000418acc  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (std::__ndk1::function<void (double)>::operator()(double) const+60) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #78 pc 000000000041881c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (reanimated::AnimationFrameCallback::onAnimationFrame(double)+48) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #79 pc 0000000000418a1c  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jni::detail::MethodWrapper<void (reanimated::AnimationFrameCallback::*)(double), &(reanimated::AnimationFrameCallback::onAnimationFrame(double)), reanimated::AnimationFrameCallback, void, double>::dispatch(facebook::jni::alias_ref<facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*>, double&&)+212) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #80 pc 0000000000418bdc  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jni::detail::CallWithJniConversions<void (*)(facebook::jni::alias_ref<facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*>, double&&), void, facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*, double>::call(facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*, double, void (*)(facebook::jni::alias_ref<facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*>, double&&))+88) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #81 pc 0000000000418888  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jni::detail::FunctionWrapper<void (*)(facebook::jni::alias_ref<facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*>, double&&), facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*, void, double>::call(_JNIEnv*, _jobject*, double, void (*)(facebook::jni::alias_ref<facebook::jni::detail::JTypeFor<facebook::jni::HybridClass<reanimated::AnimationFrameCallback, facebook::jni::detail::BaseHybridClass>::JavaPart, facebook::jni::JObject, void>::_javaobject*>, double&&))+64) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 16:56:39.386 31475-31475 DEBUG                   pid-31475                            A        #82 pc 00000000004187c0  /data/app/~~NX_UO5tDM4CIPc3TyUsx0A==/com.ecotricity-fVGe9UFXqaKYJEfUX-kEfw==/lib/arm64/libreanimated.so (facebook::jni::detail::MethodWrapper<void (reanimated::AnimationFrameCallback::*)(double), &(reanimated::AnimationFrameCallback::onAnimationFrame(double)), reanimated::AnimationFrameCallback, void, double>::call(_JNIEnv*, _jobject*, double)+60) (BuildId: 3fe38761234e4f01d2b72aed74fba84b0bcffbec)
2024-09-30 17:03:26.515 31808-31808 unknown:ReactModalHost  pid-31808                            E  Creating new dialog from context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 17:03:26.618 31808-31808 unknown:ReactModalHost  pid-31808                            E  Updating existing dialog with context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 17:03:32.772 31808-31870 ReactNativeJS           pid-31808                            E  'VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead.', '\n    at VirtualizedList (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:78791:36)\n    at FlatList (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:78095:36)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RCTScrollView\n    at ScrollView (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:69749:36)\n    at ScrollView\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at ImageBackground (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:120000:36)\n    at BackgroundImage (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:184177:24)\n    at LiveScreen (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:292540:98)\n    at StaticContainer (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:131568:17)\n    at EnsureSingleNavigator (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:127340:24)\n    at SceneView (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:131473:22)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at DebugContainer (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:204416:36)\n    at MaybeNestedStack (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:203969:23)\n    at RCTView\n    at View (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:59666:43)\n    at RNSScreen\n    at anonymous (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:75641:62)\n    at Suspender (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:206989:22)\n    at Suspense\n    at Freeze (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runModule=true:206999:23)\n    at DelayedFreeze (http://10.0.2.2:8081/index.bundle//&platform=android&dev=true&lazy=true&minify=false&app=com.ecotricity&modulesOnly=false&runMo
2024-09-30 17:03:32.941 31808-31808 unknown:ReactModalHost  pid-31808                            E  Creating new dialog from context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 17:03:33.047 31808-31808 unknown:ReactModalHost  pid-31808                            E  Updating existing dialog with context: com.ecotricity.MainActivity@5090ec@5279980
2024-09-30 17:04:03.866 31808-31808 libc                    pid-31808                            A  Fatal signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x73bbdc93e8 in tid 31808 (com.ecotricity), pid 31808 (com.ecotricity)
2024-09-30 17:04:04.913 31988-31988 DEBUG                   pid-31988                          



This is my code for using skia.

<Canvas style={{
                    flex: 1, margin: 20
                }}>

                    <Path
                        path={path}>
                        <LinearGradient
                            start={vec(0, 0)}
                            end={vec(0, 350)}
                            colors={["#3EDADA", "#B8DD22"]}
                        />
                        <CornerPathEffect r={10} />

                    </Path>

                    {currentLiveData?.consumptionFlow.gridToHome && isGridAnimation &&
                        <Path
                            path={gridToHomePath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"
                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathGridToHome} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }

                    {currentLiveData?.consumptionFlow.gridToBattery && isGridAnimation &&
                        <Path
                            path={gridToBatteryPath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"
                        >
                            <DashPathEffect intervals={[30, 2 * WIDTH + 3 * HEIGHT + 20]} phase={animatedPathGridToBattery} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }


                    {currentLiveData?.consumptionFlow.solarToHome && isSolarAnimation &&
                        <Path
                            path={solarToHomePath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"
                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathSolarToHome} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }

                    {currentLiveData?.consumptionFlow.solarToGrid && isSolarAnimation &&
                        <Path
                            path={solarToGridPath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"

                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathSolarToGrid} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }

                    {currentLiveData?.consumptionFlow.solarToBattery && isSolarAnimation &&
                        <Path
                            path={solarToBatteryPath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"

                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathSolarToBattery} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }
                    {currentLiveData?.consumptionFlow.batteryToGrid && isBatteryAnimation && <Path
                        path={batteryToGridPath}
                        style={"stroke"}
                        strokeWidth={8}
                        color={'white'}
                        strokeJoin="round"

                    >
                        <DashPathEffect intervals={[30, 250]} phase={animatedPathBatteryToGrid} />
                        <CornerPathEffect r={30} />
                    </Path>}

                    {currentLiveData?.consumptionFlow.batteryToHome && isBatteryAnimation &&
                        <Path
                            path={batteryToHomePath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"

                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathBatteryToHome} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }
                </Canvas>


Versions:
 "react": "18.2.0",
    "react-native": "0.73.4",
"@shopify/react-native-skia": "^1.3.13",

For ios it is throwing errors for some SocketRocket.

### Version

1.1.13

### Steps to reproduce

above code will help in reproduction.

### Snack, code example, screenshot, or link to a repository

<Canvas style={{
                    flex: 1, margin: 20
                }}>

                    <Path
                        path={path}>
                        <LinearGradient
                            start={vec(0, 0)}
                            end={vec(0, 350)}
                            colors={["#3EDADA", "#B8DD22"]}
                        />
                        <CornerPathEffect r={10} />

                    </Path>

                    {currentLiveData?.consumptionFlow.gridToHome && isGridAnimation &&
                        <Path
                            path={gridToHomePath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"
                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathGridToHome} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }

                    {currentLiveData?.consumptionFlow.gridToBattery && isGridAnimation &&
                        <Path
                            path={gridToBatteryPath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"
                        >
                            <DashPathEffect intervals={[30, 2 * WIDTH + 3 * HEIGHT + 20]} phase={animatedPathGridToBattery} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }


                    {currentLiveData?.consumptionFlow.solarToHome && isSolarAnimation &&
                        <Path
                            path={solarToHomePath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"
                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathSolarToHome} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }

                    {currentLiveData?.consumptionFlow.solarToGrid && isSolarAnimation &&
                        <Path
                            path={solarToGridPath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"

                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathSolarToGrid} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }

                    {currentLiveData?.consumptionFlow.solarToBattery && isSolarAnimation &&
                        <Path
                            path={solarToBatteryPath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"

                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathSolarToBattery} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }
                    {currentLiveData?.consumptionFlow.batteryToGrid && isBatteryAnimation && <Path
                        path={batteryToGridPath}
                        style={"stroke"}
                        strokeWidth={8}
                        color={'white'}
                        strokeJoin="round"

                    >
                        <DashPathEffect intervals={[30, 250]} phase={animatedPathBatteryToGrid} />
                        <CornerPathEffect r={30} />
                    </Path>}

                    {currentLiveData?.consumptionFlow.batteryToHome && isBatteryAnimation &&
                        <Path
                            path={batteryToHomePath}
                            style={"stroke"}
                            strokeWidth={8}
                            color={'white'}
                            strokeJoin="round"

                        >
                            <DashPathEffect intervals={[30, 250]} phase={animatedPathBatteryToHome} />
                            <CornerPathEffect r={30} />
                        </Path>
                    }
                </Canvas>

--- a/packages/skia/android/cpp/jni/JniSkiaManager.cpp
+++ b/packages/skia/android/cpp/jni/JniSkiaManager.cpp
@@ -9,7 +9,7 @@
 
 namespace {
 
-#ifdef REACT_NATIVE_VERSION >= 75
+#if REACT_NATIVE_VERSION >= 75
 using CallFuncType = facebook::react::CallFunc;
 #else
 using CallFuncType = std::function<void()>;

